pkg/permissions/config.go

```diff
@@ -8,11 +8,15 @@ import (
 
 // Config defines the permissions configuration structure
 type Config struct {
-	// URL is the URL checks should be executed against
+	// URL is the URL checks should be executed against.
+	// Considered unconfigured if empty.
```

Comment:
Something like this might convey intent more clearly:

```suggestion
	// URL should point to a permissions-api authorization API route, such as https://foo.dev/api/v1/allow.
	// If not set, all permissions checks will be denied by default. To override this behavior, set DefaultAllow
	// to true.
```